packages/dialect-react-ui/components/common/ThemeProvider.tsx

```diff
@@ -71,9 +71,13 @@ export type IncomingThemeValues = {
   textArea?: string;
   messageBubble?: string;
   otherMessageBubble?: string;
+  notificationBubble?: string;
+  notificationTimestamp?: string;
+  notificationsDivider?: string;
   bellButton?: string;
   iconButton?: string;
   sendButton?: string;
+  popupWrapper?: string;
```

Comment:
Let's use `modal` instead of `popup`. Is that too much work to change now?

Reply:
updated